src/ploneintranet/workspace/tests/test_content_type.py

```diff
@@ -186,3 +186,32 @@ def test_cannot_add_other_content_in_workspacecontainer(self):
             'Document',
             'doc-1',
         )
+
+    def test_attributes(self):
+        """
+        Check that we can access and set the attributes
+        """
+        self.login_as_portal_owner()
+        wsc = api.content.create(
+            self.portal,
+            'ploneintranet.workspace.workspacecontainer',
+            'workspaces',
+            title='Workspaces'
+        )
+        ws = api.content.create(
+            wsc,
+            'ploneintranet.workspace.workspacefolder',
+            'workspace-1',
+            title='Workspace 1'
+        )
+        self.assertEquals(ws.workspace_visible, True)
```

Comment:
We should use assertEqual here, as assertEquals is deprecated:
https://docs.python.org/2/library/unittest.html#deprecated-aliases